bin/_blackbox_common.sh

```diff
@@ -150,7 +158,8 @@ function prepare_keychain() {
 function add_filename_to_cryptlist() {
   # If the name is already on the list, this is a no-op.
   # However no matter what the datestamp is updated.
-  local name=$(vcs_relative_path "$1")
+  local name
+  name=$(vcs_relative_path "$1")
```

Comment:
[As suggested](https://github.com/koalaman/shellcheck/wiki/SC2155) by my linter.  updating to include comment:
>Rationale:
...the return value of `name` is ignored... will instead always return true. This may prevent conditionals, set -e and traps from working correctly.